Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:latest
```

Comment:
latest tag is not best practice to use, we require node 17. So we can  go for node:17.9.0-slim